ultralytics/solutions/streamlit_inference.py

```diff
@@ -4,145 +4,186 @@
 import time
 
 import cv2
-import torch
 
-from ultralytics.utils.checks import check_requirements
+from ultralytics.solutions.solutions import LOGGER, YOLO, check_requirements
```

Comment:
@RizwanMunawar these should be imported from their root package location, not other locations, so i.e. `from ultralytics import YOLO`.

Reply:
@glenn-jocher Hi, good catch. I am going to update it now! Thanks